doc/services/device_mgmt/smp_transport.rst

```diff
@@ -0,0 +1,208 @@
+.. _mcumgr_smp_transport_specification:
+
+SMP Transport Specification
+###########################
+
+The documents specifies information needed for implementing server and client
+side SMP transports.
+
+.. _mcumgr_smp_transport_ble:
+
+BLE (Bluetooth Low Energy)
+**************************
+
+SMP communication relays on a single GATT characteristic. An SMP command may
+be sent by using the GATT Write Without Response command.
+SMP responses are sent in the form of a GATT Notification, specifying the same
+characteristic as the GATT Write Without Response command.
+
+BLE (Bluetooth Low Energy) Characteristics
+==========================================
+
+MCUmgr Clients need to use following BLE Characteristics, when implementing
+SMP client:
+
+- **Service UUID**: `8D53DC1D-1DB7-4CD3-868B-8A527460AA84`
+- **Characteristic UUID**: `DA2E7828-FBCE-4E01-AE9E-261174997C48`
+
+All SMP communication utilizes a single GATT characteristic.  An SMP request is
+sent via a GATT Write Without Response command. An SMP response is sent in the form
+of a GATT Notification
+Response command. An SMP response is sent in the form of a GATT Notification
+specifying the same characteristic that was written.
+
+The Write Without Response form is generally preferred, as an application-layer
+response is always sent in the form of a Notification.
```

Comment:
Not valid, it is the only method, so this can go

Reply:
Gone.